cmd/dp-dimension-importer/main.go

```diff
@@ -94,7 +95,7 @@ func main() {
 		cfg.BindAddr,
 		cfg.HealthCheckInterval,
 		healthCheckErrors,
-		graphDB,
+		//graphDB,
```

Comment:
Is this needed any more? If not can we delete it.

Reply:
good catch, its still needed, i had commented it out locally but should not have committed it